java-client/src/integration-test/java/com/cumulocity/sdk/client/alarm/AlarmIT.java

```diff
@@ -309,10 +310,10 @@ public void shouldDeleteAlarmCollectionByEmptyFilter() throws Exception {
             alarmApi.create(aSampleAlarm(mo3));
         }
 
-        AlarmFilter emptyFilter = new AlarmFilter();
+        AlarmFilter requiredFilter = new AlarmFilter().byFromDate(string2Date("2000-01-01T00:00:00.000+00:00"));
```

Comment:
I've introduced required parameters as decribed here: 
as of release 10.17, at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
This change affects the following APIs:

* `DELETE /alarm/alarms` requires at least one of the following parameters: `source`, `dateFrom`, `dateTo`, `createdFrom`, `createdTo`
* `DELETE /event/events` requires at least one of the following parameters: `source`, `dateFrom`, `dateTo`, `createdFrom`, `createdTo`
* `DELETE /measurements/measurement` requires at least one of the following parameters: `source`, `dateFrom`, `dateTo`